02_join/Organisation.R

```diff
@@ -0,0 +1,35 @@
+## ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+## R code for Education Analytics Vrije Universiteit Amsterdam
+## Copyright 2024 VU
+## Web Page: http://www.vu.nl
+## Contact:
+##
+##' *INFO*:
+## 1) Join SubjectLabel and SubjectToLabel tables.
+##
+## ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+sDir <- paste0(Sys.getenv("RAW_DATA_DIR"), "Evalytics/")
+sMostrecent_zip <- vvmover::get_recent_file_date_modified(sDir, ".zip")
+
+## Read the required CSV files directly from the ZIP file
+dfOrganisation <- readr::read_delim(unz(sMostrecent_zip, "organisation.csv"))
+dfSubject <- readr::read_delim(unz(sMostrecent_zip, "subject.csv"))
+dfTeacher <- readr::read_delim(unz(sMostrecent_zip, "teacher.csv"))
+dfGenericTopic <- readr::read_delim(unz(sMostrecent_zip, "genericTopic.csv"))
+
+## Join the two data frames
```

Comment:
Is there a reason for creating three separate tables?
Can we not combine into a single table?

Reply:
There is no reason for creating three separete tables. We can indeed combine into a single table.